### PR TITLE
feat: add version tag 

### LINKED ISSUE #83: Add Version Badge and Correct API Reference Link to ReadMe
* **Add a badge that would display the current version of the SDK so as to make it visible to users**
screenshot_badge

.
.
.

* **Link the correct URL to the API reference doc**
screenshot_link

### PR DESCRIPTION
**Pull request by ChenemiKen**
- Add the SDK version tag
- fix broken link to API reference doc
fixes #83

### CODE REVIEW

**Comment by ChenemiKen** [comment]
fixed @mayorJAY

File: README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 # Novu Java SDK
 
 [![License](https://poser.pugx.org/unicodeveloper/novu/license.svg)](LICENSE.md)
+[![SDK Version](https://img.shields.io/badge/SDK-v1.4.0-blue.svg)](https://github.com/novuhq/novu-java/releases/tag/v1.4.0)
```
Comment (mayorJAY) [inline README.md:13]: Would the version change dynamically?

Comment (ChenemiKen) [reply README.md:13]: It would have to be updated when there's a new release

Comment (mayorJAY) [reply README.md:13]: A dynamic badge would be great

Comment (ChenemiKen) [reply README.md]: I have made it dynamic @mayorJAY

**Review by mayorJAY — APPROVED** [review]
LGTM 🚀

**Pull request merged**